src/command/search.rs

```diff
@@ -160,7 +160,7 @@ async fn key_handler(
             return Some(
                 app.results
                     .get(i)
-                    .map_or("".to_string(), |h| h.command.clone()),
+                    .map_or(app.input.to_string(), |h| h.command.clone()),
```

Comment:
Wait - why are you calling `.to_string` on a `String` type? :P

Reply:
Right, it needs to be cloned. I think `app.input.clone()` would be more appropriate

Reply:
Blindly replace the `""` without entirely considering the context? :eyes: never! haha thanks for the catch, sorted it